addons/web/static/src/js/libs/jquery.js

```diff
@@ -171,7 +171,7 @@ $.fn.extend({
             // Search for a body child which is at least as tall as the body
             // and which has the ability to scroll if enough content in it. If
             // found, suppose this is the top scrolling element.
-            if (el.scrollHeight < bodyHeight) {
+            if (el.getBoundingClientRect().height < bodyHeight) {
```

Comment:
For the sake of properly comparing float numbers, maybe we could go with this?
```javascript
if (bodyHeight - el.getBoundingClientRect().height > 1) { 
```
And in that case we could still use `el.scrollHeight` I guess to change as less as possible in stable versions. In that case, you can remove the MDN doc link from your commit message.